README.md

```diff
@@ -15,6 +15,7 @@
 [![DeepSource](https://static.deepsource.io/deepsource-badge-light-mini.svg)](https://deepsource.io/gh/vdaas/vald/?ref=repository-badge)
 [![Contributors](https://img.shields.io/github/contributors/vdaas/vald?style=flat-square)](https://github.com/vdaas/vald/graphs/contributors)
 [![Slack](https://img.shields.io/badge/slack-join-brightgreen?logo=slack&style=flat-square)](https://join.slack.com/t/vald-community/shared_invite/zt-db2ky9o4-R_9p2sVp8xRwztVa8gfnPA)
+[![Twitter](https://img.shields.io/badge/twitter-@vdaas__vald-blue?logo=twitter&style=flat-square)](https://twitter.com/vdaas_vald)
```

Comment:
```suggestion
[![Twitter](https://img.shields.io/badge/twitter-@vald-blue?logo=twitter&style=flat-square)](https://twitter.com/vdaas_vald)
```
I think @vald is okay, I want to use @vald account but I couldn't take it. what do you think?

Reply:
This one? https://twitter.com/vald
I think it is better to fix it after we take its account.

Reply:
no I mean I couldn't take @vald account. but vdaas_vald looks not good how about remove @ and remove vdaas_

Reply:
it's misleading for the readers.
If you don't like to use "@vdaas_vald" it's better to use the followings.
![Twitter](https://img.shields.io/badge/twitter-follow-blue?logo=twitter&style=flat-square)

Reply:
```suggestion
[![Twitter](https://img.shields.io/badge/twitter-follow-blue?logo=twitter&style=flat-square)](https://twitter.com/vdaas_vald)
```

Reply:
that's a great suggestion! LGTM